### PR TITLE
If we have a link, make it a link

### PR DESCRIPTION
**Pull request by balloob**
## Proposed change

Convert places in `<hass-tabs-subpage>` that link to other pages to use `<a>` instead of JavaScript.

(we should do this everywhere where a click handler just contains a navigate statement)

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by bramkragten** [comment]
I made this change before, but there was a reason I killed it 🤔 can't remember it now though

File: src/layouts/hass-tabs-subpage.ts [hunk 1]
```diff
@@ -88,23 +87,23 @@ class HassTabsSubpage extends LitElement {
...
-                : page.name}
-            >
-              ${page.iconPath
-                ? html`<ha-svg-icon
-                    slot="icon"
-                    .path=${page.iconPath}
-                  ></ha-svg-icon>`
-                : html`<ha-icon slot="icon" .icon=${page.icon}></ha-icon>`}
-            </ha-tab>
+            <a href=${page.path}>
```
Comment (bramkragten) [inline src/layouts/hass-tabs-subpage.ts:90]:
```suggestion
            <a href=${page.path} target=${this.supervisor ? "_parent" : "_self"}>
```

File: src/layouts/hass-tabs-subpage.ts [hunk 2]
```diff
@@ -148,6 +147,14 @@ class HassTabsSubpage extends LitElement {
                 .narrow=${this.narrow}
               ></ha-menu-button>
             `
+          : this.backPath
+          ? html`
+              <a href=${this.backPath}>
```
Comment (bramkragten) [inline src/layouts/hass-tabs-subpage.ts:152]:
```suggestion
              <a href=${this.backPath} target=${this.supervisor ? "_parent" : "_self"}>
```

File: src/layouts/hass-tabs-subpage.ts (hunk 1 above)
Comment (bramkragten) [reply src/layouts/hass-tabs-subpage.ts:90]: Or we might just want to set the name for the parent, and use that as target:

```js
window.name = "ha-main";

...

<a target="ha-main">
```

**Comment by bramkragten** [comment]
> I made this change before, but there was a reason I killed it 🤔 can't remember it now though, might have to do with the supervisor...

Yeah it was the supervisor:

image

**Comment by bramkragten** [comment]
> I made this change before, but there was a reason I killed it 🤔 can't remember it now though, might have to do with the supervisor...

Yeah it was the supervisor, it will do a full reload when we use links, we might be able to fix that though:

image

**Comment by balloob** [comment]
We just need to capture link clicks on the iframe window and convert them to navigate calls. We do this in `hc-main` with `isNavigationClick(ev)`. In that case forward it as event. I will play with it.

File: src/layouts/hass-tabs-subpage.ts (hunk 1 above)
Comment (balloob) [reply src/layouts/hass-tabs-subpage.ts:90]: I don't understand, why do we need this ?

**Review by bramkragten — APPROVED** [review]

**Pull request merged**